gno.land/cmd/gnoland/start.go

```diff
@@ -121,12 +121,11 @@ func (c *startCfg) RegisterFlags(fs *flag.FlagSet) {
 		"the root directory of the gno repository",
 	)
 
-	// XXX: Use home directory for this
 	fs.StringVar(
 		&c.dataDir,
 		"data-dir",
-		"testdir",
-		"directory for config and data",
+		"gno-chain",
```

Comment:
it's probably fine, it's better than `testdir` anyway. but I like the idea of making it explicit that we're talking about _data_, and what comes out of it (ie. this dir should not go in your git repository, and if you remove it you remove all the data of your node.)

consider `chain-data`, `node-data`, `gnoland-data`.

Reply:
I would personally vote for`gnoland-data` :)

Reply:
I like this name better too :)

[fc5f30a](https://github.com/gnolang/gno/pull/1944/commits/fc5f30a28ed06adc96f5aba1353b4528bfc8a3be)

Reply:
`.gnoland` ? 🙏🏼

Reply:
@albttx It's not something that will be located in the home folder or in a static predefined path. The default root folder is actually the current directory, not a well-known path. In its current state, I think we want to keep things explicit here so people are aware that they are creating a data directory or using the current one.
+ they can specify a specific path using a flag to explicitly target a hidden directory.